tests/marine/test_marine.py

```diff
@@ -49,8 +49,8 @@ def general_filter_and_parse_test(
         {k: str(v) for k, v in expected_output.items()} if expected_output else None
     )
 
-    assert expected_passed == passed
-    assert expected_output == output
+    assert passed == expected_passed
```

Comment:
Isn't the convention `expected == actual`? Why did you change it?

Reply:
yea i was confused too, but pycharm tells me its `acutal == excpetion`

Reply:
Use notepad++